tracing/src/main/java/com/palantir/tracing/Span.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
```

Comment:
have been tracking that for a while, but it's not done:
https://github.com/palantir/gradle-baseline/issues/59

On Thu, May 19, 2016 at 6:21 PM Mark Elliot <notifications@github.com>
wrote:

> In tracing/src/main/java/com/palantir/tracing/Span.java
> <https://github.com/palantir/http-remoting/pull/115#discussion_r63919242>:
>
> > @@ -0,0 +1,39 @@
> > +/*
> > + * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
>
> would be really nice if something enforced this...but fixing all of them
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/palantir/http-remoting/pull/115/files/cef3b7359acf8d47d417137799a5f5547358e14e#r63919242>
>